src/android/InAppBrowser.java

```diff
@@ -410,6 +425,44 @@ public void run() {
         });
     }
 
+    private void updateDialog(final String url, final boolean show) {
+        if (url == null || url.equals("") || url.equals(NULL)) {
```

Comment:
updated. I'm hoping `unHideDialog` gets dropped for `updateDialog` by you guys so the comparison can probably stay where it is for now.